Userland/Libraries/LibJS/Tests/builtins/String/String.prototype.localeCompare.js

```diff
@@ -0,0 +1,30 @@
+test("basic functionality", () => {
+    expect(String.prototype.localeCompare).toHaveLength(1);
+
+    expect("".localeCompare("")).toBe(0);
+    expect("a".localeCompare("a")).toBe(0);
+    expect("6".localeCompare("6")).toBe(0);
+
+    expect("".localeCompare("a")).not.toBe(0);
+    expect("a".localeCompare("b")).not.toBe(0);
+    expect("6".localeCompare("7")).not.toBe(0);
+    expect("a".localeCompare("A")).not.toBe(0);
```

Comment:
As mentioned before, please prefer testing exact outcomes and not just `not foo`.